gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -23,402 +30,69 @@ describe('loadSupergraphSdlFromStorage', () => {
     const result = await loadSupergraphSdlFromStorage({
       graphRef,
       apiKey,
-      endpoint: mockCloudConfigUrl,
+      endpoint: mockCloudConfigUrl1,
+      errorReportingEndpoint: mockOutOfBandReporterUrl,
       fetcher,
       compositionId: null,
+    });
+    expect(result).toMatchInlineSnapshot(sampleSchema);
```

Comment:
I might've given TMI in my original comment about this, or provided conflicting suggestions or both. It doesn't really make sense to keep this as an "inline snapshot" unless we're actually having jest write the snapshot there. Using this instead of `sampleSchema` should mean we don't need it. Sorry for the confusion there (I know one of my proposed options was to store this mock elsewhere, which is exactly what you did).
```suggestion
    expect(result).toMatchObject({
      id: 'originalId-1234',
      supergraphSdl: getTestingSupergraphSdl(),
    });
```